OloEditor/src/Panels/ContentBrowserPanel.cpp

```
@@ -20,325 +21,906 @@
+        std::string_view sv(name);
+        if (sv == "." || sv == "..")
+            return false;
+        if (sv.find('/') != std::string_view::npos || sv.find('\\') != std::string_view::npos)
+            return false;
+        return std::filesystem::path(sv).filename().string() == sv;
+    }
+} // namespace
 
+namespace OloEngine
+{
     ContentBrowserPanel::ContentBrowserPanel()
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: m_BreadcrumbTrail, m_Items, m_SelectedItems, m_RenamingItem, m_DirectoryIcon, m_FileIcon, m_ModelIcon, m_SceneIcon, m_ScriptIcon, m_AudioIcon, m_MaterialIcon, m_ShaderIcon, m_DialogueIcon, m_SaveGameIcon, m_AssetSelectedCallback

(cppcoreguidelines-pro-type-member-init)